README.md

```diff
@@ -1,2 +1,81 @@
 # Confluence
 This is Confluence of Conference.
+
+# Setup Guide
+
+### Creating a Virtual Environment
+Virtual Environment allows us to have requirements installed specifically to our project instead of complete system. To create a virtual environment goto any directory where you will store your code and run the following command on terminal (linux) or Windows PowerShell (Windows)(For windows, run Windows Powershell as Administrator before running below commands)
+```
+python -m venv venvconf
+```
+#### Activating the Virtual Environment
+Run the following command to activate your virtual environment:
+##### Windows
+```
+./venvconf/Scripts/activate
+```
+##### Linux
+```
+source venvconf/bin/activate
+```
+**Note:** To deactivate the Virtual Environment, type deactivate in the shell to deactivate.
+#### Clone your forked repo
+```
+git clone https://github.com/your_username/Confluence
+```
+##### Change to develop branch
+Code development should happed in 'develop' branch and not in 'master' branch
+```
+git checkout develop
+```
+
+#### Installing Project Requirements (First Time Only or on Requirement changes)
+Change directory to cloned directory 
+```
+cd confluence
+```
+Use pip to install  missing requirements: (if you don't have pip installed on your system, google how to install) (Windows Users should run this command Adminsitrator Shell or will get error)
+```
+pip install -r  .\requirements.txt
+```
+#### Creating Environment Variables File
+Change directory to src/confluence to create a ".env" file that will contain your enviroment variables. **Note:** This file will not ignored by git and will be available automatically on production environment.
```

Comment:
`.env` does not get included in git. That is the whole point of keeping settings as environment variables (because we want to prevent including confidential information on git tree)